lib/spack/spack/test/buildrequest.py

```diff
@@ -3,12 +3,21 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+import sys
+
 import pytest
 
 import spack.installer as inst
 import spack.repo
 import spack.spec
 
+# Spack features encapsulated by this test are
+# supported on Windows but are currently
+# non functional due to Windows
+# compatibility issues.
```

Comment:
I'm sorry, for whatever reason, a take away from an earlier conversation lead me to believe we wanted to phrase it in this structure, but I'm realizing on revisit that I misread. Basically this functionality should, and will eventually, work on Windows, however it currently does not because we have not yet gone through and made it Windows compatible. Essentially Spack on Windows will support this, but for the time being does not.

Reply:
Got it - thanks! I would prefer if this comment use that wording.

Reply:
Oh me too, unsure why I was so fixed on the previous structure... Glad its resolved, sorry about that.

Reply:
This appears to be the same

Reply:
Ping